docs/assets/0123-import-scripts/import-script.py

```diff
@@ -0,0 +1,161 @@
+import os
+import warnings
+
+import pandas as pd
+import regex
+
+from knora import excel2xml
+
+# general preparation
+# -------------------
+path_to_json = "import-project.json"
+main_df = pd.read_csv("data-raw.csv", dtype="str", sep=",")  # or: pd.read_excel("*.xls(x)", dtype="str")
+
+# remove rows without usable values (prevents Errors when there are empty rows at the end of the file)
+main_df = main_df.applymap(lambda x: x if pd.notna(x) and regex.search(r"[\p{L}\d_!?]", str(x), flags=regex.U) else pd.NA)
```

Comment:
a lambda with a ternary expression containing a regex is rather advanced python - again, depending on the target audience, this may not be the most understandable line of code

Reply:
IMO, they are not required to understand every detail, but this is a template that they can start with. I'm aware that the average RDU member wouldn't have written lines 15-16 himself, but that's the reason that I have written them for him. 

I had quickly thought about hiding these two lines in a method, in order to hide away complexity. But then I decided to leave them here, so that the RDU members can learn from it.